### PR TITLE
Write server logs to file

### LINKED ISSUE #7: Write stdout/stderr logs from server to a file
I'm trying to work out why a conformance test is failing for .NET, and I *suspect* the server log would tell me - but I can't see that log.

If the validation framework could write to serverlog-stdout.txt and serverlog-stderr.txt or something similar, that would be wonderful.

### PR DESCRIPTION
**Pull request by juliehockett**
Fixes #7

### CODE REVIEW

File: client/main.go [hunk 1]
```diff
@@ -48,6 +48,7 @@ func runValidation() error {
 func main() {
 	flag.Parse()
 	if err := runValidation(); err != nil {
+		log.Printf("%v", err)
```
Comment (hdp617) [inline client/main.go:51]: nit: this is equivalent to `log.Fatalf("%v", err)`.

**Comment by grant** [comment]
@juliehockett Mergable?

**Comment by grant** [comment]
@googlebot: rescan

**Comment by grant** [comment]
CLA should be fine. Bot is acting up.

**Pull request merged**